operational/itam/schedule_itam_report/schedule-itam-report.pt

```diff
@@ -0,0 +1,192 @@
+        formattedName = formattedName.slice(2); // Remove the first two elements Result: [UserToAssignedAsset, AssetID]
+      }
+      formattedName = formattedName.join(" / "); // Join the remaining column name values with a ` / `. Result: "UserToAssignedAsset / AssetID"
+      formattedName = formattedName.replace(/([A-Z])+/g, " $&"); // Add space before each sequential series of capital letter. Result: "User To Assigned Asset / Asset ID"
+      formattedName = formattedName.charAt(0).toUpperCase() + formattedName.slice(1);
+      formattedName = formattedName.trim();
+      return formattedName;
+    }
+
+    //get the header columns
+    result.columns = [];
+    // Use first row to get the column names
+    // This assumes that all rows have the same columns
+    // Put object keys and labels into an ordered array to make the output consistent
+    _.each( _.keys(data[0]).sort(), function(key) {
+      var label = humanizeColumnName(key);
+      result.columns.push({
+        // key is used to get the value from the row
+        "key": key,
+        // label is used in the output report as a column header
+        "label": label,
+        // label_pad is used for the second row of the markdown table to create a separator
+        "label_pad": pad('-',label.length)
+      });
+    });
+    // Begin constructing the markdown table
+    // sResult is the markdown table as a string
+    result.sResult = "";
+
+    // First line of a markdown table has the column header names
+    // | column1 | column2 | column3 | ... |
+    result.sResult += "| " + _.pluck(result.columns, 'label').join(' | ').trim() + " |\\n";
+    // Second line of a markdown table has the column header separators
+    // | ------- | ------- | ------- | ... |
+    result.sResult += "| " + _.pluck(result.columns, 'label_pad').join(' | ') + " |\\n";
+
+    // Now the rest of the table the data
+    _.each(data, function(line) {
+      // sLine holds the string for the current row in the markdown table
+      // Start row with `| `
+      sLine = "| ";
+      // For each column, use the key to get the value from the row
+      // This is needed to prevent printing the values of object in different order than the column headers
+      _.each(result.columns, function(column) {
+        // Append to sLine the value from the row and column and a closing ` | `
+        sLine += line[column.key] + " | ";
+      });
+      // Trim the trailing whitespace and add a newline to finish the row
+      result.sResult += sLine.trim() + "\\n";
+    });
+  EOS
+end
+
+policy 'ds_fnms_reports_combined' do
+  validate $ds_markdown_report do
+    summary_template "Flexera ITAM Report - {{data.name}}"
+    detail_template <<-EOS
+{{data.sResult}}
+    EOS
+    escalate $send_report
+    check eq(1, 0) # The check always fails which results in the escalation always triggering
+  end
+end
+
+escalation "send_report" do
```

Comment:
Use prefix `esc_` for escalation declarations

Reply:
Fixed in https://github.com/flexera-public/policy_templates/pull/1226/commits/1d884c09d545674513d743ac53271dd1f0c0e544